conf/base.py

```diff
@@ -152,6 +152,14 @@
     'TEST_REQUEST_DEFAULT_FORMAT': 'vnd.api+json',
     'DEFAULT_VERSIONING_CLASS': 'rest_framework.versioning.URLPathVersioning',
     'COERCE_DECIMAL_TO_STRING': False,
+    'DEFAULT_THROTTLE_CLASSES': (
+        "apps.throttles.OrgBurst",
```

Comment:
I don't think we need to track both burst/sustained. Really (for billing) we only care about monthly calls - see if monthly is possible, otherwise we can use a daily value that corresponds to the monthly value.

Reply:
SGTM